draft-irtf-cfrg-frost.md

```diff
@@ -879,6 +821,11 @@ to the description in {{dep-pog}}. Validation steps for these functions
 are described for each the ciphersuites below. Future ciphersuites MUST
 describe how input validation is done for DeserializeElement and DeserializeScalar.
 
+We include explicit `verify()` functions for each ciphersuite that should also
+apply to any singleton signature as well as signatures generated by FROST. By
+definition, this function assumes that all group Elements passed as input,
+including the signature R component and public key, belong to prime-order group G.
```

Comment:
```suggestion
We include explicit `verify()` functions for each ciphersuite. Since FROST-generated
signatures and singleton-generated Schnorr signatures are identical, these functions
verify both. By definition, this function assumes that all group Elements passed as
input, including the signature R component and public key, belong to prime-order
group G.
```

Reply:
Either phrasing works for me.